data/json/recipes/other/tool.json

```diff
@@ -177,5 +177,49 @@
     "book_learn": [ [ "mag_survival", 2 ], [ "textbook_survival", 1 ], [ "cookbook", 2 ] ],
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "can_food_unsealed", 2 ], [ "scrap", 4 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "hand_axe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "survival",
+    "skills_required": [ "fabrication", 1 ],
+    "difficulty": 3,
+    "time": 54000,
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ [ "rock", 1 ], [ "ceramic_shard", 1 ], [ "sharp_rock", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
```

Comment:
This makes more sense as a metal analogue of the stone axe, in other words just a chunk of metal with a sharp part usable for limited cutting activities.

Remove the haft and the increased combat stats, and make its tool qualities match the stone axe.